tests/foreman/cli/test_satellitesync.py

```diff
@@ -981,7 +981,8 @@ def test_negative_export_cv_with_on_demand_repo(self):
             })
         self.assert_error_msg(
             error,
-            'All exported repositories must be set to an immediate download policy and re-synced'
+            "Could not export the content view:\n  Error: Ensure the content view version"
+            " {} has at least one repository.\n".format(cv_dict.values())[1]
```

Comment:
Are you sure about this indexing? It takes second character from created string. It makes more sense to me to take second value from dict, but you also need to wrap `cv_dict.values()` in `list()` for that.

```python
>>> cv_dict = {'one': 1, 'two': 2}
>>> "Could not export the content view:\n  Error: Ensure the content view version {} has at least one repository.\n".format(cv_dict.values())[1]
'o'
>>> "Could not export the content view:\n  Error: Ensure the content view version {} has at least one repository.\n".format(cv_dict.values()[1])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'dict_values' object does not support indexing
>>> "Could not export the content view:\n  Error: Ensure the content view version {} has at least one repository.\n".format(list(cv_dict.values())[1])
'Could not export the content view:\n  Error: Ensure the content view version 2 has at least one repository.\n'
```

Reply:
@mirekdlugosz, I see what you mean and have updated the code.  So this should be up to standard now.